spec/services/idv/profile_maker_spec.rb

```diff
@@ -32,7 +33,7 @@
       expect(profile.encrypted_pii).to_not be_nil
       expect(profile.encrypted_pii).to_not match('Some')
       expect(profile.fraud_pending_reason).to be_nil
-      expect(profile.proofing_components).to match(proofing_component.as_json)
+      expect(profile.proofing_components).to match(proofing_components.to_json)
```

Comment:
I think we want to undo this too
```suggestion
      expect(profile.proofing_components).to match(proofing_components.as_json)
```